examples/torch/sac_half_cheetah.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+"""
+This is an example to train a task with DDPG algorithm written in PyTorch.
+
+Here it creates a gym environment InvertedDoublePendulum. And uses a DDPG with
+1M steps.
+
+"""
+import gym
+import torch
+from torch.nn import functional as F  # NOQA
+from torch import nn as nn
+
+from garage.envs import normalize
+from garage.envs.base import GarageEnv
+from garage.experiment import LocalRunner, run_experiment
+from garage.replay_buffer import SimpleReplayBuffer
+from garage.torch.algos import SAC
+from garage.torch.policies import TanhGaussianMLPPolicy
+from garage.torch.q_functions import ContinuousMLPQFunction
+
+
+def run_task(snapshot_config, *_):
+    """Set up environment and algorithm and run the task."""
+    runner = LocalRunner(snapshot_config)
+    env = GarageEnv(normalize(gym.make('HalfCheetah-v2')))
+
+    policy = TanhGaussianMLPPolicy(env_spec=env.spec,
+                               hidden_sizes=[256, 256],
+                               hidden_nonlinearity=nn.ReLU,
+                               output_nonlinearity=None)
+
+    qf1 = ContinuousMLPQFunction(env_spec=env.spec,
+                                hidden_sizes=[256, 256],
+                                hidden_nonlinearity=F.relu)
+
+    qf2 = ContinuousMLPQFunction(env_spec=env.spec,
+                                hidden_sizes=[256, 256],
+                                hidden_nonlinearity=F.relu)
+
+    target_qf1 = ContinuousMLPQFunction(env_spec=env.spec,
```

Comment:
same story -- if these must be the same model class then you should `copy.deepcopy()` them internally.

Reply:
I'm not sure if that actually works though, since these networks need to have different initializations.

Also, AFAICT, there's technically no requirement that these be the same architecture?